.github/workflows/build_docs.yaml

```diff
@@ -1,12 +1,12 @@
 name: Build and Upload Docs
 
-on:
+"on":
   push:
+    tags:
+      - "*"
```

Comment:
Does this mean we automatically push on all tags? I think we do not want this: we should only push on tags that correspond to releases, for example ones that follow the semantic versioning format exactly. If we make tags like "for_testing_in_foo_conditions", we don't want it built or distributed on pip or conda. Even in the case of "alpha" versions, I don't think we want these distributed that way: only production versions should show up in pypi or conda.

Reply:
Yes, this pushes on all tags. 
Tags with 'alpha', 'beta' or 'rc' can be recognized by PyPI and will result in releases there -- other labels would fail. (a 'dev' tag, for example would fail).

Reply:
Oh, but also - these are docs. I'm actually not concerned about docs being published that don't correspond to full production release tags. There are already docs which are published for *every* PR -- this is actually really useful so that you can make changes to the documentation and see what actually would turn up.